build.gradle.kts

```
@@ -7,28 +7,19 @@ buildscript {
 }
 
 plugins {
-    kotlin("multiplatform") version "1.4.31" apply false
-    kotlin("js") version "1.4.31" apply false
-    kotlin("plugin.serialization") version "1.4.31" apply false
+    kotlin("multiplatform") version "1.5.10" apply false
+    kotlin("js") version "1.5.10" apply false
+    kotlin("plugin.serialization") version "1.5.10" apply false
     id("org.jmailen.kotlinter") version "3.2.0" apply false
     id("com.vanniktech.maven.publish") version "0.14.0" apply false
-    id("org.jetbrains.dokka") version "1.4.30"
+    id("org.jetbrains.dokka") version "1.4.32"
     id("binary-compatibility-validator") version "0.2.3"
```

Any chance you can bump this to 0.6.0 (assuming it doesn't break anything)?

```suggestion
    id("binary-compatibility-validator") version "0.6.0"
```

Seeing a lot of changes in `koap.api` and wondering if it is just an incompatibility of `binary-compatibility-validator` w/ the newer Kotlin compiler. Or perhaps the new compiler is showing us that we had some function with too public of visibility.

---

I noticed that as well. Let me give this a try and see what happens

---

Those functions are marked `internal` unless I'm misreading that output. Perhaps adding `explicitApi()` will help?